src/appState.ts

```diff
@@ -3,7 +3,7 @@ import { getDateTime } from "./utils";
 
 export const DEFAULT_FONT = "20px Virgil";
 
-export function getDefaultAppState(): AppState {
```

Comment:
I think these are better in cases like this

Reply:
Define better.. I was thinking about transforming them all to arrow functions..

Reply:
Cases where you want a named function with a block body make more sense to me as a `function`.

Reply:
Yea, named functions are better for debugging (stack traces).

Reply:
I don’t think that matters as much because modern browsers will properly name functions, and minification changes the names anyway.

Reply:
@j-f1 True. So what was your argument for keeping `function` (apart from the refactor being unnecessary)?

Reply:
I just liked it better subjectively, which isn’t a very good reason.